python/vyos/frrender.py

```diff
@@ -233,6 +233,17 @@ def dict_helper_nhrp_defaults(nhrp):
             ip_dict['afi'] = ip_version
             dict.update({ip_version : ip_dict})
 
+    # Get FRR profile
+    frr_system_cli_path = ['system', 'frr']
+    dict['frr'] = {
+        'defaults': conf.get_config_dict(
+            frr_system_cli_path,
+            key_mangling=('-', '_'),
+            get_first_key=True,
+            with_recursive_defaults=True,
+        )['profile']
+    }
```

Comment:
Why so complicated in addressing an inner key of the dict? Just use the already existing patterns - this makes the life a lot easier for the maintainers if any change adhere to already existing design / coding rules.

```suggestion
    dict['system_frr'] = conf.get_config_dict(frr_system_cli_path , key_mangling=('-', '_'),
                                              get_first_key=True, with_recursive_defaults=True)    
```